.github/actions/feedstock-required-pr/action.yml

```diff
@@ -7,6 +7,19 @@ description: "Required global PR tests for Open-CE feedstocks."
 runs:
   using: "composite"
   steps:
-    - name: PR Test Placeholder
+    - name: Install dependencies
       shell: bash -l {0}
-      run: echo "No required tests added yet."
+      run: |
+        conda install -y conda-build
+    - name: Check Dependencies
+      shell: bash -l {0}
+      run: |
+        # Validate all env files that use the current feedstock.
+        open_ce_path="$GITHUB_ACTION_PATH/../../../open-ce"
+        env_path="$GITHUB_ACTION_PATH/../../../envs"
+        feedstock=$(echo ${PWD##*/} | sed 's|\(.*\)-.*|\1|')
+        for f in ${env_path}/*.yaml; do
+          if grep -Fq "${feedstock}" ${f}; then
```

Comment:
I don't think this will handle nested environment files properly. Maybe we just want to do this on the open-ce.yaml file if we are worried about build time.

Reply:
I was more wanting to make sure that it ran on everything that used it. In case it's a feedstock outside of open-ce.
I could switch it to test all env files. I don't think we're really worried about time.

Reply:
@bnemanich 
Okay, it's checking all env files now for the default variants and cuda+cpu.